### PR TITLE
Fixes #9829 - Revert image_search to old method

### PR DESCRIPTION
**Pull request by eLobato**

### CODE REVIEW

File: app/models/foreman_docker/docker.rb [hunk 1]
```diff
@@ -69,7 +69,7 @@ def tags(image_name)
     end
 
     def search(term = '')
-      ::Docker::Image.search({ :term => term }, docker_connection)
+       client.images.image_search(:term => term)
```
Comment (daviddavis) [inline app/models/foreman_docker/docker.rb]: Unnecessary whitespace at start of line.

:dog2:ci

**Comment by eLobato** [comment]
@daviddavis updated

**Comment by daviddavis** [comment]
@eLobato, this works. Would be nice to have a test :smiley_cat: but ACK otherwise.

**Comment by daviddavis** [comment]
Merged as 2472bb7.

**Pull request closed without merging**